### ISSUE TITLE
Map and Battle scenes drawing delay of official corescript ver1.6.1

### ISSUE THREAD

**Issue by rev2nym** [rpgtkoolmv/corescript#180]
In official corescript ver1.6.1,
Drawing map and battle scenes is delay at scene starting.
Background image is drawn first, and tile images are drawn after it.
The cause is to begin drawing by Spriteset_Base before finishing loading image files.

It's possible to check by following way.
It's also possible to test on a Community edition corescript.

Apply this code and run as test.
It will display "false" at first calling.
```
var _Spriteset_Base_update =
    Spriteset_Base.prototype.update;
Spriteset_Base.prototype.update = function() {

    console.log(ImageManager.isReady());

    _Spriteset_Base_update.call(this);
};
```

And try following code.
This causes a stack overflow in map scene.
```
var _Spriteset_Base_update =
    Spriteset_Base.prototype.update;
Spriteset_Base.prototype.update = function() {

    console.log(SceneManager._scene.isReady());

    _Spriteset_Base_update.call(this);
};
```

These mean that sprites have begun to draw though images aren't ready.

**Comment by rev2nym** [comment]
I abandoned large modifications to maintain compatibility.
I removed the access to $gameSystem by Scene_Base for the time being.

**Comment by krmbn0576** [comment]
Close this because it is solved by #194
